Modules/Session/classes/class.ilObjSession.php

```diff
@@ -849,10 +862,14 @@ public function getTutorSource() {
 	public function getParentCourseTutors() {
 		global $tree;
 		$tutors = array();
-		$parent = array(
-			'type'=>'none',
-			'ref_id' => $this->getRefId()
-		);
+
+		$ref_id = $this->getRefId();
+		//during creation:
+		if(! $ref_id) {
+			$ref_id = $_GET['ref_id'];
+		}
```

Comment:
@klees Ich bin mir nicht ganz sicher ob das hier nicht zu Problemen führen kann. Ist die ref id in der url beim Erstellen nicht immer die vom Parent?

Reply:
Ja, genau das ist beabsichtigt.
es geht dann weiter: 
$parent = $tree->getNodeData($ref_id);
while($parent['type'] !== 'crs') { ....

Reply:
Imo correct. Der baut dann ja erst das neue Objekt. Wird ansonsten auch beim Testing auffallen.